services/src/main/java/org/exoplatform/addons/gamification/service/configuration/RealizationsServiceImpl.java

```diff
@@ -67,4 +118,97 @@ public GamificationActionsHistoryDTO updateRealizationStatus(Long gHistoryId,
     gHistory.setStatus(status.name());
     return realizationsStorage.updateRealizationStatus(gHistory);
   }
+  
+  private boolean isAdministrator(Identity identity) {
```

Comment:
private methods should goes at the end of the file, see standard java class elements ordering.

Reply:
Okay I will .